index.html

```diff
@@ -528,5 +528,17 @@ <h2>ts-help-text</h2>
 			</ts-aside>
 			<ts-cover id="my-cover"></ts-cover>
 		</ts-root>
+		<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (11) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)